README.md

```diff
@@ -1,17 +1,36 @@
 # Entropy Tunkki
 
-### dev
+### initialize environment
 * copy .env.example to .env and symfony/.env to /symfony/.env.dev.local and change the defaults
 * `docker compose build; docker compose up -d;`
 
-#### install assets
-* docker compose run --rm node yarn install
-* docker compose run --rm node yarn build
 #### draw the rest of the owl
 * docker compose exec fpm composer install
+
+#### database restore from dump
 * docker compose exec fpm ./bin/console doctrine:database:import dump.sql
-or create tables
+
+#### database creation
 * docker compose exec fpm ./bin/console doctrine:schema:update --force
+
+#### install assets
+* docker compose run --rm node yarn install
+* docker compose run --rm node yarn build
+
+### Access tunkki
+* open http://localhost:9090/ in your browser
+
+### Initial creation of new user and setting it as super admin
+* fill in the registration form at http://localhost:9090/profile/new and submit.
```

Comment:
I already fixed the console command. It can be now used to create the user too.

Reply:
https://github.com/EntropyRy/Tunkki/blob/symfony6/symfony/src/Command/UserCommand.php#L45

Reply:
When I run
docker compose exec fpm ./bin/console entropy:member --create-user teemu@tapanila.net --super-admin

I get following errors:
In ExceptionConverter.php line 49:
                                                                                                                                        
  An exception occurred while executing a query: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.member' doesn't exist  
                                                                                                                                        

In Exception.php line 28:
                                                                                         
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.member' doesn't exist  
                                                                                         

In Statement.php line 130:
                                                                                         
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'test.member' doesn't exist

Reply:
Just something wrong with my setup. Nevermind this error